lib/Doctrine/ORM/Mapping/ClassMetadataInfo.php

```diff
@@ -1584,7 +1586,7 @@ private function isTypedProperty(string $name): bool
      *
      * @param  array{fieldName: string, type?: mixed} $mapping The field mapping to validate & complete.
```

Comment:
if we update return type, we should update the `type?: string` here in param too